javascript/app-discover/src/components/panels/VariablePropertiesPanel.tsx

```diff
@@ -48,6 +48,7 @@ export const VariablePropertiesPanel: React.FC<VariablePropertiesPanelProps> =
 		const [inModelVariables, setInModelVariables] = useRecoilState(
 			InModelCausalVariablesState,
 		)
+		const varrr = inModelVariables?.map((v) => v.columnName)
```

Comment:
maybe 'varNames'?

Reply:
Yep, I totally forgot about this varr that I created to test. Thanks!